.golangci.yaml

```diff
@@ -68,7 +73,6 @@ linters:
     - nakedret
     - nolintlint
     - prealloc
-    - revive
```

Comment:
revive seems to be quite versatile yet hard to configure. IIUC it seems the linter used to be not disabled and suddenly started outputting a huge number of errors after I updated the golangci-lint. I've tried fixing them, but gave it up and just disabled it for now...

Reply:
Created a different issue for revive https://github.com/external-secrets/external-secrets/issues/3397